rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -85,6 +85,13 @@ def ensure_global(name, default):
         # TIP: if using the sphinx-bootstrap-theme, you need
         # "treeViewIsBootstrap": True,
         "exhaleExecutesDoxygen": False,
+        # Maps markdown files to the "md" lexer, and not the "markdown" lexer
+        # Pygments registers "md" as a valid markdown lexer, and not "markdown"
+        "lexerMapping": {{r".*\.md": "md",}},
+        # This mapping will work when `exhale` supports `:doxygenpage:` directives
+        # Check `svenevs/exhale#111`
```

Comment:
I suggest using the full GitHub url here, or mention it is on GitHub, otherwise you don't know where it is hosted, could be GitLab for instance.

Reply:
Full link added in 2317689.